tests/zfs-tests/tests/functional/delegate/delegate_common.kshlib

```diff
@@ -425,6 +428,49 @@ function verify_fs_receive
 	return 0
 }
 
+function verify_fs_receive_append
+{
+	typeset user=$1
+	typeset perm=$2
+	typeset fs=$3
+
+	typeset dtst
+	typeset stamp=${perm}.${user}.$RANDOM
+	typeset newfs=$fs/newfs.$stamp
+	typeset bak_user=$TEST_BASE_DIR/bak.$user.$stamp
+
+	log_must zfs create $newfs
+	typeset datasets="$newfs"
+
+	for dtst in $datasets ; do
```

Comment:
The test you copied this from was having this loop to test also zvol.  You removed the zvol, but left the loop.

Reply:
Right, I am going to remove the loop. Thanks.